sdk/src/commonMain/kotlin/io/timemates/sdk/users/profile/ProfileApi.kt

```diff
@@ -47,4 +48,9 @@ public class ProfileApi(
         return tokenProvider.getAsResult()
             .flatMap { token -> engine.execute(EditProfileRequest(token, null, name, description)) }
     }
+
+    public suspend fun setGravatar(emailAddress: EmailAddress): Result<Empty> {
+        val setGravatarRequest = SetGravatarRequest(emailAddress)
```

Comment:
you're missing tokenProvider usage, also in your request there should be authorization present. Take a look at other methods inside this API.